src/main/groovy/wooga/gradle/upm/artifactory/UPMArtifactoryConventions.groovy

```diff
@@ -2,6 +2,9 @@ package wooga.gradle.upm.artifactory
 
 import com.wooga.gradle.PropertyLookup
 import com.wooga.gradle.PropertyUtils
+import org.gradle.api.Project
+import org.gradle.api.file.Directory
+import org.gradle.api.provider.Provider
```

Comment:
Why all these imports all of a sudden?

Reply:
sorry this was an in-between thing, i initially put the convenience function in here, then I realized this is no place for logic, only for conventions, removed the function, but not he imports :)